### DISCUSSION TITLE
Release 0.11.0 with proper Quarkus 2.4 support?

### DISCUSSION THREAD

**Discussion by famod** [quarkiverse/quarkus-cxf#307] [General]
Hi everyone,

I saw #301 and was asking myself, whether it would make sense to cut a new release with "proper" Quarkus 2.4 support?

Having said that, my current app (mutliple SOAP clients, no provided SOAP endpoints) seems to be working fine with 0.10.0 and Quarkus 2.4.1.

**Comment by shumonsharif** [comment]
Hey @famod Glad to hear that you're having success with 0.10.0 and Quarkus 2.4.1.  Are you using native mode; the issue should only occur during the native build?

I think we should definitely cut a new release as you suggest; though I was hoping to close https://github.com/quarkiverse/quarkus-cxf/issues/278#issuecomment-953226225 prior to releasing.

**Reply by famod** [reply]
No, not native...which makes sense then!

**Comment by shumonsharif** [comment]
@famod @vsevel @darkeagle1337 Release 0.11.0 with Quarkus 2.5 support is out!
